postfix/datadog_checks/postfix/data/conf.yaml.example

```diff
@@ -57,3 +57,21 @@ instances:
     tags:
       - optional_tag3
       - optional_tag4
+
+
+## Log Section (Available for Agent >=6.0)
+##
+## type - mandatory - Type of log input source (tcp / udp / file / windows_event)
+## port / path / channel - mandatory - Set port if type is tcp or udp. Set path if type is file and channel if windows_event
+## service - mandatory - Name of the service owning the log
+## source  - mandatory - Attribute that defines which integration is sending the logs
+## sourcecategory - optional - Multiple value attribute. Can be used to refine the source attribtue
```

Comment:
"Can be used", implying that it this attr can do other things, too. What other things?

Reply:
right now it's used just for this.

Reply:
Feel free to remove it, as we plan to change the behaviour for this and it seems to not bring much clarity anywhere.

Reply:
Removed